composer.json

```diff
@@ -49,7 +54,5 @@
     },
     "config": {
         "preferred-install": "dist"
-    },
-    "minimum-stability": "stable",
-    "prefer-stable": true
```

Comment:
Why were these two lines removed? While this is a shits and giggles project, wanna make sure we are using stable versions of packages and such. Don't want to have Dashboarders bitching at us due to stuff suddenly broken lol